BUILD

```diff
@@ -113,7 +113,7 @@ g_stands_for = "goat"  # @unused
 
 core_version = "18.0.0"  # @unused
```

Comment:
Does the core_version need to be bumped too?

Reply:
There is no API change.  The documentation says then we bump the minor version but in practice I think we only ever bump them major version when there is an API change, so I left it the same way as our other releases. let me know if this is ok

Reply:
Sounds good.